src/css/main.css

```diff
@@ -7,1987 +7,2034 @@
+             letter-spacing: 0.5px;
+             font-weight: 400;
+             text-transform: capitalize;
+         }
+         .list-group-item-active h5 strong,
+         .list-group-item-active h5 i{
+             color: white !important;
+         }
+         .list-group-item-active h5 i{
+             position:relative;
+             top: 0px !important;
+         }
+         .list-group-item h5 i,
+         .list-group-item-active h5 i {
+             width: 20px !important;
+             transition: 0.1s;
+             padding: 0;
+             text-align: right !important;
+         }
+         .list-group-item h5 i:hover,
+         .list-group-item-active h5 i:{
+             cursor: pointer;
+             opacity: 0.5;
+             transition: 0.1s;
+         }
+ 
+     .list-group-item .fa,
+     .list-group-item-active .fa{
+         width: 25px !important;
+     }
+ 
+ .sidebar-footer{
+     display: block;
+     box-sizing: border-box;
+     padding: 15px !important;  
+     text-align: center;
+     border-top: 1px dashed rgba(0,0,0,0.25);
+     margin-top: 55px; 
+ }
+ .sidebar-footer a{
+     font-weight: 400;
+ }
+ 
+ 
+ /*
+     Toast
+     Toast
+     Toast
+ */
+ /* .toast {
+     background-color: #0F8AFF !important;
+     color: #fff !important;
+     position: fixed;
+     bottom: 15px !important;
+     top: auto !important;
+     right: 10px !important;
+     left: auto !important;
+     text-align: left !important;
+     font-weight: 300 !important;
+     letter-spacing: 0.5px !important;
+     line-height: 18px !important;
+     padding-left: 60px !important;
+     box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24) !important;
+ }
+ .toast .toast-title{
+     font-weight: 400 !important;
+     font-size: 14px !important;
+ }
+ .toast .toast-title:after {
+     display: inline-block;
+     float: right;
+     content: '\f12a';
+     font-size: 12px;
+     font-family: "Ionicons";
+     color: #FFF;
+     position: relative;
+     top: -5px;
+     left: 5px;
+     opacity: 0.5;
+     transition: 0.1s;
+ }
+ .toast:after{
+     display: none !important;
+     box-shadow: none !important;
+ }
+ .toast:hover {
+     box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23) !important;
+ }
+ .toast:hover .toast-title:after {
+     opacity: 1;
+     transition: 0.1s;
+ } */
+ 
+ 
+ /*
+     Modals
+ */
+ #aboutTabs{
+     padding: 0 !important;
+ }
+ 
+ 
+ /*
+     Report
+     Report
+     Report
+ */
+ #printArea h3 {
+     font-size: 20px !important;
+     padding: 0 0 0 3px !important;
+     margin-bottom: 15px !important;
+ }
+ #printArea .leaflet-control-layers-base input[type=radio]{
+     -webkit-appearance: radio !important;
+     appearance: radio !important;
+     width: 20px !important;
+     display: inline !important;
+ }
+ 
+ .disclaimerReport p {
+     max-width: 100%;
+     margin-bottom: auto;
+     padding-top: 5px;
+ }
+ 
+ #printArea span:not(rzslider span) {

```

Comment:
We use `printArea` in other places so we just want to avoid any unknown consequences from editing this. I think I checked most of them & it was fine, I guess I am just wondering what this change was for?

I think it used to be: 
```
#printArea span {
padding: 0 0 0 3px !important;
}
```

<!--EndFragment-->
</body>
</html>